4.3/README.md

```diff
@@ -117,6 +127,89 @@ docker run -it --shm-size=1g \
   --name nominatim \
   mediagis/nominatim:4.3
 ```
+### Configuration Example
+
+A setup example with almost all Flag possibilities including a short explanation:
+
+```sh
+docker run -it \
+
+	-v nominatim-flatnode:/nominatim/flatnode \
+#Sets the flatnode file, which is to reduce the load on the database when you plan to use multiple countrys together bigger than 6GB
```

Comment:
Please move all the comments _above_ the line they are commenting and indent them to the same level as the code.

Reply:
done!